app/views/admin/payments/index.html.haml

```diff
@@ -0,0 +1,27 @@
+%section.admin_payments
+  .container
+    .row
+      .col-sm-10.col-sm-offset-1
+        %section.payment_history
+          %ul.nav.nav-tabs
+            %li.active
+              %a(href="") Recent Payments
+            %li
+              = link_to "Add a New video", new_admin_video_path
+          %table.table
+            %thead
+              %tr
+                %th Name
+                %th Email
+                %th Amount
+                %th Reference ID
+            %tbody
+              - @payments.each do |payment|
+                %tr
+                  %td
+                    = payment.user.full_name
+                  %td
+                    = payment.user.email
+                  %td "$9.99"
```

Comment:
I would suggest using a value from the database and not hard coding into the view.